### PR TITLE
Fix/cov

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | no 
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no
| Tickets       | fixes #... 
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -120,7 +120,7 @@ jobs:
           composer update --no-progress --no-suggest --ansi
       - name: Install PHPUnit
         env:
-          SYMFONY_PHPUNIT_VERSION: '9.2'
+          SYMFONY_PHPUNIT_VERSION: '8.3'
```
Comment (dunglas) [inline .github/workflows/ci.yml:123]: It's better to run PHPStan with the latest versions of all deps.

File: src/Bridge/Symfony/Bundle/Test/Constraint/ArraySubset.php [hunk 2]
```diff
@@ -13,94 +13,15 @@
 
 namespace ApiPlatform\Core\Bridge\Symfony\Bundle\Test\Constraint;
 
-use PHPUnit\Framework\Constraint\Constraint;
 use PHPUnit\SebastianBergmann\Comparator\ComparisonFailure;
 use SebastianBergmann\Comparator\ComparisonFailure as LegacyComparisonFailure;
 
 if (!class_exists(ComparisonFailure::class)) {
-    class_alias(LegacyComparisonFailure::class, ComparisonFailure::class);
+    class_alias(LegacyComparisonFailure::class, 'PHPUnit\SebastianBergmann\Comparator\ComparisonFailure');
```
Comment (dunglas) [inline src/Bridge/Symfony/Bundle/Test/Constraint/ArraySubset.php:20]: You can keep using ::class even if the class doesn't exist.

File: src/Bridge/Symfony/Bundle/Test/Constraint/ArraySubsetV9.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ * subject array.
+ *
+ * Imported from dms/phpunit-arraysubset-asserts, because the original constraint has been deprecated.
+ *
+ * @copyright Sebastian Bergmann <sebastian@phpunit.de>
+ * @copyright Rafael Dohms <rdohms@gmail.com>
+ *
+ * @see https://github.com/sebastianbergmann/phpunit/issues/3494
+ */
+final class ArraySubsetV9 extends Constraint
```
Comment (dunglas) [inline src/Bridge/Symfony/Bundle/Test/Constraint/ArraySubsetV9.php:31]: Mark these classes as internal maybe?

File: src/Bridge/Symfony/Bundle/Test/Constraint/ArraySubset.php (hunk 2 above)
Comment (soyuka) [reply src/Bridge/Symfony/Bundle/Test/Constraint/ArraySubset.php:20]: phpstan does not agree

Comment (soyuka) [reply src/Bridge/Symfony/Bundle/Test/Constraint/ArraySubset.php:20]: When using `::class`, phpstan loads the class and fails because it doesn't find it.

**Comment by dunglas** [comment]
Thanks @soyuka!

**Pull request merged**